src/swsssdk/scripts/sonic-db-cli

```diff
@@ -1,38 +1,51 @@
-    print >> sys.stderr, msg
-else:
-    dbname = sys.argv[1]
-    dbconn = swsssdk.SonicV2Connector(use_unix_socket_path=False)
-    try:
-        dbconn.connect(dbname)
-    except RuntimeError:
-        msg = "Invalid database name input : '{}'".format(sys.argv[1])
-        print >> sys.stderr, msg
-    else:
-        client = dbconn.get_redis_client(dbname)
-        args = sys.argv[2:]
-        resp = client.execute_command(*args)
-        """
-        sonic-db-cli output format mimic the non-tty mode output format from redis-cli
-        based on our usage in SONiC, None and list type output from python API needs to be modified
-        with these changes, it is enough for us to mimic redis-cli in SONiC so far since no application uses tty mode redis-cli output
-        """
-        if resp is None:
-            print ""
-        elif isinstance(resp, list):
-            print "\n".join(resp)
+'''
+  Examples of how to use this script 
+      Example 1: sonic-db-cli -ns asic0 CONFIG_DB keys *
+      Example 2: sonic-db-cli -ns asic1 APPL_DB HGETALL VLAN_TABLE:Vlan10
+      Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu
+      Example 4: sonic-db-cli -ns asic5 APPL_DB EVAL "return {KEYS[1],KEYS[2],ARGV[1],ARGV[2]}" 2 k1 k2 v1 v2
+'''
+
+def main():
+    parser = argparse.ArgumentParser(description='SONiC DB CLI',
+                         version='1.0.0',
+                         formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument('-ns', '--namespace', type=str, help='Namespace to use [asic0,asic1...,asicn]', default='default')
+    parser.add_argument('database', nargs='?', type=str, help='Database name')
+    parser.add_argument('cmd', nargs='+', type=str, help='Command to execute in database')
+
+    args = parser.parse_args()
+
+    if args.database and args.cmd:
+        dbconn = swsssdk.SonicV2Connector()
+        try:
+            dbconn.connect(args.database, namespace=args.namespace)
+        except RuntimeError:
+            msg = "Invalid database name input : '{}'".format(args.database)
+            print >> sys.stderr, msg
         else:
-            print resp
+            client = dbconn.get_redis_client(args.database)
+            cmdStr = ' '.join([str(elem) for elem in args.cmd])
+            resp = client.execute_command(cmdStr)
+            """
+            sonic-db-cli output format mimic the non-tty mode output format from redis-cli
+            based on our usage in SONiC, None and list type output from python API needs to be modified
+            with these changes, it is enough for us to mimic redis-cli in SONiC so far since no application uses tty mode redis-cli output
+            """
+            if resp is None:
+                print ""
+            elif isinstance(resp, list):
+                print "\n".join(resp)
+            else:
+                print resp
+    else:
+        parser.print_help()
+        sys.exit(1)
```

Comment:
Add new testcase for new code

Reply:
Updated test/test_moduleLoad.py for handling the database_global.json file and namespace changes to SonicDBConfig class. 

I have verified the sonic-db-cli by making sure the docker services are up on a DUT with  these changes. If we need to add specific py testcases, we would need to use something like mock redis client.